addon/components/g-map.js

```diff
@@ -118,5 +144,43 @@ export default Ember.Component.extend({
   groupMarkerClicked(marker, group) {
     let markers = this.get('markers').without(marker).filterBy('group', group);
     markers.forEach((marker) => marker.closeInfowindow());
+  },
+
+  attachEvents() {
+    const events = this.get('supportedEvents');
+
+    for (let i = 0; i < events.length; i++) {
+      const event = events[i];
+      if (isPresent(this.get(event))) {
+        this.addListener(event);
+      }
+    }
+  },
+
+  addListener(event) {
+    const map = this.get('map');
+
+    if (isPresent(map)) {
+      google.maps.event.addListener(map, event, () => {
+        const action = this.get(event);
```

Comment:
shouldn't it be `this.get(`attrs.${event}`);` for closure actions? (I'm not sure, just asking)